serverlessworkflow/modules/ROOT/pages/cloud/enhanced-service-discovery.adoc

```diff
@@ -0,0 +1,128 @@
+= Enhanced Service Discovery
+:compat-mode!:
+// Metadata:
+:description: Explain what is and how the service discovery works
+:keywords: kogito, workflow, quarkus, serverless, service-discovery, enhanced-service-discovery
+// links
+:quarkus_issue_url: https://github.com/quarkusio/quarkus/issues/27457
+
+
+The Enhanced Service Discovery allows you to have a static URI that will define the Kubernetes resource that you use
+to perform http requests, the following sketch demonstrate how the SWUrl is composed:
```

Comment:
what is SWUrl ? Can we call it `Kubernetes Serverless Workflow URI` ?

Reply:
Please call it just `URI`.